pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+### Description
+<!--
+_Using one or more sentences, describe the proposed changes and the reason for making them._
+-->
+
+### Related JIRA Issue(s)
+<!--_Please provide the URL(s) for any JIRA issues related to this PR._-->
+
+### Review App URL(s) 
+http://__BRANCH_NAME_HERE__.review.ensembl.org
+
+
+### Knowledge Base
+<!--
+_If there is any new concept which are being implemented then provide link(s) to the Knowledge base (Link to Documentation, Blog etc )_
+-->
+
+### Example(s)
+<!--
+_Bacckedn we mostly work with APIs so add request/response of the API call or How to navigate in front end app (How one can view particular track which uses the changes in the PR) etc . or anything relevant information which allows reviewer to view your changes in action_
```

Comment:
`Bacckedn` -> `Backend`? Maybe drop this entirely, and compress the comment to `Any details that will help your colleagues to review the PR, such as the expected request/response, instructions for viewing the changes via the web client, or any other relevant information`.